demo/chatbot_demo/run.py

```diff
@@ -17,11 +17,12 @@ def chat(message, history):
     return history, history
 
 
+state = gr.Variable(value=[])
 chatbot = gr.Chatbot(color_map=("green", "pink")).style()
```

Comment:
On the other hand, `color_map` is not part of the constructor for `gr.Chatbot`, it's been moved to `style()`